primer/index.bs

```diff
@@ -915,6 +916,8 @@ Notice that the `keys` field is an array, so an OP could have multiple public ke
 Using the `kid` value in the ID token, the AS searches the OP public keys for one that matches the
 one used to sign the ID token. If no public key is found, the AS must reject the request.
 
+Details about the `kid` can be found in the [RFC7515 section 4.1.4](https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.4).
```

Comment:
Ordinarily, with Bikeshed, you would refer to an RFC like so:

`[[RFC7515#section-4.1.4]]`

(For normative references, you would include an exclamation point `[[!RFC7515]]`, but for the primer, everything should be informative).

Reply:
OK thanks @acoburn. I just changed it in the [last commit](https://github.com/solid/solid-oidc/pull/230/commits/deb20ba003b7685058e116f005a97a2d1fcd0f3c).